Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -637,6 +639,12 @@ def _on_loaded(tdef):
         deferred.addCallback(_on_loaded)
         return deferred
 
+    def start_download_from_infohash(self, infohash, dconfig=None):
```

Comment:
You can also turn the infohash to a magnet link using `"magnet:?xt=urn:btih:" + infohash` and then start a magnetlink download.